test/FourBitTwoDisclosureDeviceUnlockerTest.java

```diff
@@ -541,5 +606,9 @@ public void testValidDoPokeCalls() throws Exception {
         doPoke.invoke(FourBitTwoDisclosureDeviceUnlocker.class);
         state = (int) stateField.get(FourBitTwoDisclosureDeviceUnlocker.class);
         Assert.assertTrue(state == statePoked);
+
```

Comment:
A lot of testing methods that were using reflection to ensure bad input to result in defined behavior were causing other tests to fail.
For example, one test would change the field `changeBitTo` to an invalid character like `'G'`. Then other tests would fail because they would attempt to change bits to `'G'`.

This fixes that by resetting them back to default values. I did not care to make them private access again, however.

Reply:
Sorry about that! I didn't even think to change the access, or changing it back to the original value.

Reply:
It's alright, I always forget doing this when mocking in test suites designed for JavaScript.